contracts/SampleToken.sol

```diff
@@ -27,17 +28,8 @@ contract SampleToken is MintableToken {
     public
     returns (bool)
   {
-    require(jurisdiction.hasAttribute(_to, "VALID"));
+    require(validator.transferAllowed(msg.sender, _to, _amount));
```

Comment:
This is not clear in the TPL spec, but I think we should be checking `transferAllowed(0x0, _to, _amount)`, i.e. `from` = `0x0` which is usually how `minting` is thought of.